app/models/miq_worker/service_worker.rb

```diff
@@ -17,6 +17,13 @@ def create_container_objects
         container[:env] << {:name => "PORT", :value => container_port.to_s}
         container[:env] << {:name => "BINDING_ADDRESS", :value => "0.0.0.0"}
         add_readiness_probe(container)
+        container[:volumeMounts] ||= []
+        definition[:spec][:template][:spec][:volumes] ||= []
+
+        if kind_of?(MiqUiWorker)
+          container[:volumeMounts] << {:name => "ui-httpd-configs", :mountPath => "/etc/httpd/conf.d"}
+          definition[:spec][:template][:spec][:volumes] << {:name=>"ui-httpd-configs", :configMap=>{:name=>"ui-httpd-configs", :defaultMode=>420}}
+        end
```

Comment:
Can this be possibly be in the MiqUiWorker itself?  Two ways I can see doing it are

1. Have a `create_container_objects` method in MiqUiWorker that calls super.
2. Have this part call a method if responds_to? then only implement that method in MiqUiWorker

Reply:
> 1. Have a `create_container_objects` method in MiqUiWorker that calls super.

I like this version.  We might need to move the block contents into a method `build_deployment_definition` or something so we can override and call super.  Naming is hard though.